### PR TITLE
Desktop-version added to the My-portfolio-site using media quires 

### PR DESCRIPTION
**Pull request by emhamza**
The desktop version has been added to my portfolio website which has been built by following the one the Figma templates. While writing the code for this version I have used media queries of the mobile-first approach as well as desktop-first approach together. I also made a breakpoint at the width of 768px in my portfolio website.

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -43,133 +57,152 @@ <h1 class="item2">I'm a software developer</h1>
...
       
-      <ul class="abilites">
+      <ul class="project-abilites">
         <li class="a1">Ruby on rails</li>
         <li class="a1">css</li>
         <li class="a1">JavScript</li>
         <li class="a1">html</li>          
-      </ul>
-
-      <div class="b_div"><button class="button">See Project</button></div>
```
Comment (AdnanOlarmmi) [inline portfolio.html:70]: - [ ] Please, kindly fix the vertical spacing between these cards, to enable a very pixel-perfect design, like you'll be doing in your forthcoming job 😀.

- Your design
wrong

- Figma design
right

File: portfolio.html [hunk 2]
```diff
@@ -43,133 +57,152 @@ <h1 class="item2">I'm a software developer</h1>
...
+      
+        <div class="b_div">
+          <button class="see-button">
+            See Project
+          </button>
+        </div>
       </div>
-
-   </section>
-   <section about_me_container>
```
Comment (AdnanOlarmmi) [inline portfolio.html:172]: - [ ] Please, kindly fix the vertical spacing between these cards, to enable a very pixel-perfect design, like you'll be doing in your forthcoming job 😀.

- Your design
wrong1

- Figma design
right1

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @emhamza, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Professional Readme.
- Added test.md file

## Required Changes ♻️
- Please fix the margins in your design to match the design, to enable you have a very pixel-perfect design 🙃.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @emhamza , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
imageswelljob
## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Good looking UI :art: 
- [ ] Descriptive PR `title` and `summary` :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**